From e2d051e302713362e153ec40da5f74a81746dea5 Mon Sep 17 00:00:00 2001
From: Doug Ledford <dledford@redhat.com>
Date: Wed, 24 Sep 2014 14:58:23 -0400
Subject: [infiniband] usnic: Set userspace/kernel ABI ver to 4

Message-id: <eb4bb278d49fb7e38493479a6fc1b9d5609c7eab.1411569661.git.dledford@redhat.com>
Patchwork-id: 96595
O-Subject: [Patch RHEL7 21/29] IB/usnic: Set userspace/kernel ABI ver to 4
Bugzilla: 916382
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Upinder Malhi <umalhi@cisco.com>

usNIC userspace/kernel ABI should be set to 4 instead of 3.

Signed-off-by: Upinder Malhi <umalhi@cisco.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit 6dcebe614c667fca73aaf0cfbd1e70bc9179538e)
Signed-off-by: Doug Ledford <dledford@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/usnic/usnic_abi.h b/drivers/infiniband/hw/usnic/usnic_abi.h
index 014bcfc..04a6622 100644
--- a/drivers/infiniband/hw/usnic/usnic_abi.h
+++ b/drivers/infiniband/hw/usnic/usnic_abi.h
@@ -21,7 +21,7 @@
 #define USNIC_ABI_H
 
 /* ABI between userspace and kernel */
-#define USNIC_UVERBS_ABI_VERSION 3
+#define USNIC_UVERBS_ABI_VERSION 4
 
 #define USNIC_QP_GRP_MAX_WQS  8
 #define USNIC_QP_GRP_MAX_RQS  8
-- 
1.7.1